### PR TITLE
Azure: add README for the root module

### PR DESCRIPTION
**Pull request by diogocp**

### CODE REVIEW

**Review by mariagardbraga — APPROVED** [review]

**Review by jinroh — APPROVED** [review]

File: modules/azure/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+| [azapi_resource_id.api_key_id](https://registry.terraform.io/providers/Azure/azapi/1.13.1/docs/data-sources/resource_id) | data source |
+| [azapi_resource_id.key_vault_id](https://registry.terraform.io/providers/Azure/azapi/1.13.1/docs/data-sources/resource_id) | data source |
+| [azurerm_key_vault.key_vault](https://registry.terraform.io/providers/hashicorp/azurerm/3.101.0/docs/data-sources/key_vault) | data source |
+| [azurerm_subscription.current](https://registry.terraform.io/providers/hashicorp/azurerm/3.101.0/docs/data-sources/subscription) | data source |
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| <a name="input_admin_ssh_key"></a> [admin\_ssh\_key](#input\_admin\_ssh\_key) | SSH public key of the admin user | `string` | n/a | yes |
```
Comment (jhgilbert) [inline modules/azure/README.md:41]:
```suggestion
| <a name="input_admin_ssh_key"></a> [admin\_ssh\_key](#input\_admin\_ssh\_key) | SSH public key of the admin user. | `string` | n/a | yes |
```

File: modules/azure/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+| <a name="input_api_key"></a> [api\_key](#input\_api\_key) | Specifies the API key required by the Agentless Scanner to submit vulnerabilities to Datadog | `string` | `null` | no |
```
Comment (jhgilbert) [inline modules/azure/README.md:42]:
```suggestion
| <a name="input_api_key"></a> [api\_key](#input\_api\_key) | Specifies the API key required by the Agentless Scanner to submit vulnerabilities to Datadog. | `string` | `null` | no |
```

File: modules/azure/README.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+| <a name="input_api_key_secret_id"></a> [api\_key\_secret\_id](#input\_api\_key\_secret\_id) | The versionless resource ID of the Azure Key Vault secret holding the Datadog API key. Ignored if api\_key is specified. | `string` | `null` | no |
+| <a name="input_bastion"></a> [bastion](#input\_bastion) | Create a bastion in the subnet | `bool` | `false` | no |
```
Comment (jhgilbert) [inline modules/azure/README.md:44]:
```suggestion
| <a name="input_bastion"></a> [bastion](#input\_bastion) | Create a bastion in the subnet. | `bool` | `false` | no |
```

File: modules/azure/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+| <a name="input_create_roles"></a> [create\_roles](#input\_create\_roles) | Specifies whether to create the role definitions and assignments required to scan resources. | `bool` | `true` | no |
+| <a name="input_location"></a> [location](#input\_location) | The location where the Datadog Agentless Scanner resources will be created | `string` | n/a | yes |
```
Comment (jhgilbert) [inline modules/azure/README.md:46]:
```suggestion
| <a name="input_location"></a> [location](#input\_location) | The location where the Datadog Agentless Scanner resources will be created. | `string` | n/a | yes |
```

File: modules/azure/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+| <a name="input_resource_group_name"></a> [resource\_group\_name](#input\_resource\_group\_name) | The name of the resource group where the Datadog Agentless Scanner resources will be created | `string` | n/a | yes |
```
Comment (jhgilbert) [inline modules/azure/README.md:47]:
```suggestion
| <a name="input_resource_group_name"></a> [resource\_group\_name](#input\_resource\_group\_name) | The name of the resource group where the Datadog Agentless Scanner resources will be created. | `string` | n/a | yes |
```

File: modules/azure/README.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+| <a name="input_scan_scopes"></a> [scan\_scopes](#input\_scan\_scopes) | The set of scopes that the Datadog Agentless Scanner is allowed to scan. Defaults to the current subscription. | `list(string)` | `[]` | no |
+| <a name="input_site"></a> [site](#input\_site) | By default the Agent sends its data to Datadog US site. If your organization is on another site, you must update it. See https://docs.datadoghq.com/getting_started/site/ | `string` | `null` | no |
+| <a name="input_tags"></a> [tags](#input\_tags) | A map of additional tags to add to the resources created | `map(string)` | `{}` | no |
```
Comment (jhgilbert) [inline modules/azure/README.md:50]:
```suggestion
| <a name="input_tags"></a> [tags](#input\_tags) | A map of additional tags to add to the resources created. | `map(string)` | `{}` | no |
```

File: modules/azure/README.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| <a name="output_vnet"></a> [vnet](#output\_vnet) | The Azure Virtual Network created for the Datadog agentless scanner |
```
Comment (jhgilbert) [inline modules/azure/README.md:56]:
```suggestion
| <a name="output_vnet"></a> [vnet](#output\_vnet) | The Azure Virtual Network created for the Datadog agentless scanner. |
```

**Review by jhgilbert — APPROVED** [review]
Approved with very minor suggestions to make the punctuation consistent, thanks!

**Pull request merged**